doc/api/stream.md

```diff
@@ -1500,13 +1500,12 @@ buffered. When the `callback` is invoked, the stream might emit a [`'drain'`][]
 event. If a stream implementation is capable of processing multiple chunks of
 data at once, the `writable._writev()` method should be implemented.
 
-If the `decodeStrings` property is set in the constructor options, then
-`chunk` may be a string rather than a Buffer, and `encoding` will
-indicate the character encoding of the string. This is to support
-implementations that have an optimized handling for certain string
-data encodings. If the `decodeStrings` property is explicitly set to `false`,
-the `encoding` argument can be safely ignored, and `chunk` will remain the same
-object that is passed to `.write()`.
+If the `decodeStrings` property is explicitly set to `false` in the constructor
+options, then `chunk` will remain the same object that is passed to `.write()`,
+and may be a string rather than a Buffer. This is to support implementations
```

Comment:
s/Buffer/`Buffer` (surround with backticks) ... this should be consistent throughout the doc if it's not already.